### PR TITLE
feat: dont print escape backslash on strings containing escaped backslashes

### LINKED ISSUE #378: Backslashes escaped in object values
I am logging [polylines](https://developers.google.com/maps/documentation/javascript/examples/polyline-simple) which include the character `\`. When I try to log them, it escapes the backslash as it's JSON encoding the value. e.g.

```
pino.info({ polyline: '\\' })
```

outputs `polyline: "\\"`, when it would be more correct to output `polyline: "\"`

An extra `\` can mess up the shape of the polyline e.g.

```
{o_uIzhvS?P_@VcAr@u@j@[\\QDKAUQISb@sDAe@Ai@G[GCWC_@KSIWSAQ
```

```
{o_uIzhvS?P_@VcAr@u@j@[\QDKAUQISb@sDAe@Ai@G[GCWC_@KSIWSAQ
```

Thanks :)

### PR DESCRIPTION
**Pull request by ademarsj**
Closes #378 , I'm not pretty sure about the way that I made it, but I think creating a function to do exactly that line of code is unnecessary.

### CODE REVIEW

**Comment by mcollina** [comment]
Could you fix it in Node v14? Otherwise it would need to wait until we bump the major to drop node v14.

**Comment by ademarsj** [comment]
> 

Working on that.

**Comment by ademarsj** [comment]
Done, passing through all tests.

**Review by mcollina — APPROVED** [review]
lgtm

File: lib/utils.js [hunk 1]
```diff
@@ -404,16 +404,20 @@ function prettifyObject ({
     if (Object.keys(plain).length > 0) {
       result += colorizer.greyMessage(stringifySafe(plain))
     }
+    // Avoid printing the escape character on escapting backslash
     result += eol
+    result = result.replace(/\\\\/gi, '\\')
   } else {
```
Comment (jsumners) [inline lib/utils.js:410]:
```suggestion
    result += eol
    // Avoid printing the escape character on escaped backslashes.
    result = result.replace(/\\\\/gi, '\\')
  } else {
```

**Review by jsumners — CHANGES_REQUESTED** [review]

**Comment by ademarsj** [comment]
Grammar issues fixed.

**Review by jsumners — APPROVED** [review]

**Comment by ntzm** [comment]
Thanks 🎉

**Pull request merged**